server/src/data.source.ts

```diff
@@ -9,8 +9,8 @@ export const dbDataSource: DataSourceOptions = {
   password: 'root',
   database: 'todo',
   entities: [__dirname + '/**/*.entity.ts', __dirname + '/**/*.entity.js'],
-  migrationsRun: false,
-  migrations: [__dirname + '/migration/*.ts'],
+  migrationsRun: true,
+  migrations: [__dirname + '/migration/*.ts', __dirname + '/migration/*.js'],
```

Comment:
This works, but can you try
```suggestion
  migrations: [__dirname + '/migration/*{.ts,.js}'],
```
to see if it works?
If so, please also update the regex for entities as well!

Reply:
Thanks for the feedback! 
It actually worked! 
Will commit again :)